app.js

```diff
@@ -2,13 +2,13 @@ require('dotenv').config()
 const express = require('express')
 const cors = require('cors')
 const app = express()
-const port = 3000
+const port = process.env.PORT || 3000
 app.use(express.json())
 app.use(cors())
 //All endpoints go here
-const router = require('../job-culture-api-Backend/routes/userrouter')
-const companyrouter = require('../job-culture-api-Backend/routes/companyrouter')
-const reviewrouter = require('../job-culture-api-Backend/routes/reviewrouter')
+const router = require('./routes/userrouter')
```

Comment:
fixed wrong import. heroku does git-checkout for a random/custom folder, and it is better to only import from the root level. `./`